ethereum/1inch/b2c.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "1inch",
+    "contracts": [
+        {
+            "contractName": "1inchV3",
+            "address": "0x11111112542D85B3EF69AE05771c2dCCff4fAa26",
+            "selectors": {
+                "0x7c025200": {
+                    "method": "swap",
+                    "plugin": "1inch",
+                    "erc20OfInterest": []
```

Comment:
We need to tell ledgerjs in which field of the tx we expect to find the erc20 address. Have a look at `aave`, `lido`, and `paraswap` :)